.github/workflows/general_handle-stale-issues-and-prs.yml

```
@@ -0,0 +1,34 @@
+name: 'Close stale issues and PRs'
+on:
+  workflow_call:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      # only needed for delete-branch option (which we are currently not using)
+      # contents: write
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
```

Per https://github.com/mui/mui-public/security/code-scanning/9, I imagine it should be

```suggestion
      - uses: actions/stale@5bef64f19d7facfb25b37b414482c7164d639639 # v9
```